website/screens/components/date-input/code/examples/basic.ts

```diff
@@ -1,25 +1,10 @@
 import { DxcDateInput, DxcStack, DxcInset } from "@dxc-technology/halstack-react";
 
 const code = `() => {
-    return(
-        <DxcInset space="large">
-            <DxcStack gutter="small">
-                <DxcDateInput
-                label="Default"
-                placeholder="Placeholder"
-                />
-                <DxcDateInput
-                label="Error"
-                placeholder="Placeholder"
-                error="Error message"
-                />
-                <DxcDateInput
-                label="Disabled"
-                placeholder="Placeholder"
-                disabled
-                />
-            </DxcStack>
-        </DxcInset>
+  return (
+    <DxcInset space="large">
+        <DxcDateInput label="Default" placeholder="Placeholder" />
```

Comment:
I suggest adding size="fillParent" to the example to improve the look of the component:

![image](https://user-images.githubusercontent.com/44321109/171350033-7ecb482e-2b2c-4065-bccc-74151348f668.png)

Reply:
After the poll, we need to do the change to have every example look the same.